template/docs/SUPPORT.md

```diff
@@ -0,0 +1,25 @@
+# Getting Support
+
+Zend Framework offers three support channels:
+
+- For real-time questions, use our
+  [Slack](https://zendframework-slack.herokuapp.com)
+- For detailed questions (e.g., those requiring examples) use our
+  [forums](https://discourse.zendframework.com/c/contributors)
```

Comment:
I think the link here should depend on the package, and for expressive packages it should be:
https://discourse.zendframework.com/c/questions/expressive
and for other components:
https://discourse.zendframework.com/c/questions/components

Reply:
Done.